data/src/main/java/camp/nextstep/edu/github/data/GitHubService.kt

```diff
@@ -0,0 +1,22 @@
+package camp.nextstep.edu.github.data
+
+import camp.nextstep.edu.github.domain.GithubRepository
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.http.GET
+
+internal interface GitHubService {
+    @GET("repositories")
+    fun listRepos(): Call<List<RepoModel>>
```

Comment:
```suggestion
    suspend fun listRepos():List<RepoModel>
```
Retrofit 사용 시 suspend 키워드를 붙여주면 내부적으로 코루틴을 지원해요.